src/dos/dos.cpp

```diff
@@ -178,6 +178,22 @@ void DOS_SetCountry(uint16_t country_number)
 	dos.tables.country[DOS_DECIMAL_SEPARATOR_OFS]   = country_info.decimal_separator;
 }
 
+uint16_t DOS_GetCurPackedTime()
+{
+	uint32_t ticks   = mem_readd(BIOS_TIMER);
+	uint32_t seconds = (ticks * 10) / 182;
+	uint16_t hour    = (uint16_t)(seconds / 3600);
+	uint16_t min     = (uint16_t)((seconds % 3600) / 60);
+	uint16_t sec     = (uint16_t)(seconds % 60);
+
+	return DOS_PackTime(hour, min, sec);
+}
+
+uint16_t DOS_GetCurPackedDate()
```

Comment:
`DOS_GetCurrentPackedDate` — it's not like those few extra keystrokes cost money 😁